openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/expansion-panel/_expansion-panel-theme.scss

```
@@ -0,0 +1,19 @@
+@mixin expansion-panel-theme($theme){
+
+  $app-primary: mat-color(map-get($theme, primary));
+  $app-accent: mat-color(map-get($theme, orders));
```

for the generic component this accent color should probably be 
```suggestion
$app-accent: mat-color(map-get($theme, accent));
```

---

Yep good call